src/css/my-catalog.css

```diff
@@ -136,6 +157,12 @@
 }
 
 @media (min-width: 769px) {
```

Comment:
min 768px, a jak coś ma być tylko w tablecie a na desktop nie to musisz dodać jeszcze max 1279